### PR TITLE
Admin dashboard backend

### PR DESCRIPTION
**Pull request by trevorwinser**
- Remade admin_test.py
- Reconfigured branch to include zone in DB creation.
- Added zone to table in admin dashboard

### CODE REVIEW

**Review by danglakshay — APPROVED** [review]
Code looks good! Approved!

File: code/templates/admin/admin-dashboard.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
                     <th id="password">Password</th>
                     <th id="email">Email</th>
                     <th id="phone">Phone</th>
+                    <th id="alerts_email">Zone</th>
```
Comment (noahstaz) [inline code/templates/admin/admin-dashboard.html:29]: Why does this share an id with the line below it?

**Review by noahstaz — APPROVED** [review]
Everything looks good to merge to me, just take a look at the one comment first :)

File: code/templates/admin/admin-dashboard.html (hunk 1 above)
Comment (trevorwinser) [reply code/templates/admin/admin-dashboard.html:29]: Accident! The zone field didn't exist so I did a hasty Ctrl C, Ctrl V

**Pull request merged**